packages/perseus/src/server-item-renderer.tsx

```diff
@@ -2,12 +2,12 @@
 import * as PerseusLinter from "@khanacademy/perseus-linter";
 import {StyleSheet, css} from "aphrodite";
 /**
- * A copy of the ItemRenderer which renders its question renderer and hints
- * renderer normally instead of ReactDOM.render()ing them into elements in the
- * DOM.
+ * The main item (aka Exercise) renderer in Perseus. This component renders its
+ * question renderer and hints renderer using standard React practices (an
+ * earlier ItemRenderer component used to use jQuery and ReactDOM.render()ing
+ * to render into elements it didn't own.
  *
- * This allows this component to be used in server-rendering of a perseus
- * exercise.
+ * This component is compatible with server-rendering of a perseus exercise.
```

Comment:
This was done just to trigger an npm snapshot. For some reason that I couldn't figure out the Github Action to publish a snapshot was being skipped until I modified source code.  🤷‍♂️